client/angular/controllers/MainController.js

```diff
@@ -193,6 +208,9 @@
             } else {
                 $scope.displayColumns = sortedColumns;
             }
+            $scope.onscreenTweets = (backfilledColumns.reduce(function(prevColumn, curColumn) {
```

Comment:
Should this be done with displayColumns rather than backfilledColumns? It seems to be intended to be the list of all tweets that are being displayed, but with the way displayColumns is used it would be the right choice for that I think